emails/en/sucessful-password-reset.pug

```diff
@@ -0,0 +1,6 @@
+extends index
+block title
+    h1 Hi, #{firstName}
+block description
+    p The password for your SpaceToStudy account was changed. If you did not change it, please visit the main page and reset it yourself
```

Comment:
why main page? why can't just redirect it directly to the reset password page?

Reply:
Because we do not have a reset password page.

Reply:
where a person is supposed to reset password then?